### PR TITLE
[UX] Consistent wizard message texts

### PR DESCRIPTION
**Pull request by kthoms**
This will make the texts in the new project wizard more consistent.

- Sentences finished by dot.
- Renamed WizardNewXtextProjectCreationPage_LabelFacets to
AdvancedNewProjectPage_LabelFacets since it belongs to the advanced
page.
- Renamed 'Plugin' to 'Plug-in'. This is how it is spelled in Eclipse
wording.
- Renamed 'eclipse' to 'Eclipse'.

### CODE REVIEW

File: org.eclipse.xtext.xtext.ui/src/org/eclipse/xtext/xtext/ui/wizard/project/messages.properties [hunk 1]
```diff
@@ -8,37 +8,37 @@
 ###############################################################################
 
 NewXtextProjectWizard_WindowTitle=New Xtext Project
-WizardNewXtextProjectCreationPage_Description=Click 'Finish' to create a basic Xtext Project or 'Next' to go to the advanced configuration
+WizardNewXtextProjectCreationPage_Description=Click 'Finish' to create a basic Xtext project or 'Next' to go to the advanced configuration.
```
Comment (cdietrich) [inline org.eclipse.xtext.xtext.ui/src/org/eclipse/xtext/xtext/ui/wizard/project/messages.properties]: is still inconsistent case wise

File: org.eclipse.xtext.xtext.ui/src/org/eclipse/xtext/xtext/ui/wizard/project/messages.properties [hunk 2]
```diff
@@ -8,37 +8,37 @@
...
 XtextProjectCreator_CreatingProjectsMessage2=Creating DSL Projects 
 AdvancedNewProjectPage_WindowTitle=Advanced Xtext Configuration
-AdvancedNewProjectPage_Description=Customize your Xtext Project
-AdvancedNewProjectPage_eclipseAndGradleWarn=Building Eclipse Plugins with Gradle is not yet supported. An additional Maven Tycho build will be created
-AdvancedNewProjectPage_ideaAndMavenWarn=Building IntelliJ Plugins with Maven is not yet supported. An additional Gradle build will be created
-AdvancedNewProjectPage_p2AndSdkInfo=The creation of an update site will automatically create a feature project
-AdvancedNewProjectPage_ideaReqGradleInfo=IntelliJ Plugin requires Gradle build. An additional Gradle build will be created
-AdvancedNewProjectPage_noBuildship=Gradle integration for eclipse is not installed. Consider to install Buildship.
-AdvancedNewProjectPage_noM2e=Maven integration for eclipse is not installed. Consider to install M2e.
+AdvancedNewProjectPage_Description=Customize your Xtext Project.
```
Comment (svenefftinge) [inline org.eclipse.xtext.xtext.ui/src/org/eclipse/xtext/xtext/ui/wizard/project/messages.properties]: project should be lower case

File: org.eclipse.xtext.xtext.ui/src/org/eclipse/xtext/xtext/ui/wizard/project/messages.properties [hunk 3]
```diff
@@ -8,37 +8,37 @@
...
+AdvancedNewProjectPage_LabelFacets=Facets
+AdvancedNewProjectPage_eclipseAndGradleWarn=Building Eclipse Plug-ins with Gradle is not yet supported. An additional Maven Tycho build will be created.
```
Comment (svenefftinge) [inline org.eclipse.xtext.xtext.ui/src/org/eclipse/xtext/xtext/ui/wizard/project/messages.properties]: plug-ins should be lower case

**Comment by kthoms** [comment]
@svenefftinge @cdietrich Is it OK now?

**Pull request merged**